config-windows.md

```diff
@@ -94,3 +94,42 @@ The following parameters can be specified:
         }
    }
 ```
+
+## <a name="configWindowsCredentialSpec" />Credential Spec
+
+You can configure a container's group Managed Service Accounts (gMSAs) via the OPTIONAL `credentialspec` field of the Windows configuration. For more information about gMSAs, see [Active Directory Service Accounts for Windows Containers][gMSAOverview]. For more information about tooling to generate a gMSA, see [Deployment Overview][gMSATooling]. The `credentialspec` MUST be a string containing an escaped JSON object.
```

Comment:
Instead of making this a string with serialized JSON, can you make it a JSON object, say the object properties are implementation-defined, and use an `interface{}` in the Go type?  That should get you the same flexibility while still letting you (de)serialize in a single pass.